apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -131,6 +131,27 @@ window.PLOrderBlocks = function (uuid, options) {
       placePairingIndicators();
       setAnswer();
     },
+    start: function () {
+      if (enableIndentation) {
+        $(dropzoneElementId)[0].style.background =
+          'linear-gradient(#000, #000) no-repeat border-box, '.repeat(+maxIndent + 1).slice(0, -2);
+        $(dropzoneElementId)[0].style.backgroundSize = '1px 100%, '
+          .repeat(+maxIndent + 1)
+          .slice(0, -2);
+        $(dropzoneElementId)[0].style.backgroundPosition = (function () {
+          let counter = 0;
+          let pos_string = '';
+          while (counter <= maxIndent) {
+            pos_string += `${
+              +$(dropzoneElementId).css('padding-left').slice(0, -2) + TABWIDTH * counter
+            }px 0, `;
+            counter += 1;
+          }
+          console.log(pos_string);
+          return pos_string.slice(0, -2);
+        })();
```

Comment:
We can definitely get this code much shorter and easier to read. Maybe something like 
```
$(dropzoneElementId)[0].style.backgroundPosition = Array.from({length: maxIndent}, (value, index => {
  `$(dropzoneElementId).css('padding-left').slice(0, -2) + TABWIDTH * counter}px 0`
}).join(' , ')
```
(I haven't tested this, it might need to be tweaked a little).

Reply:
Your code works, just needed to use `maxIndent + 1` in the length to account for the default indent: for future reference in case someone digs this up 5 years from now, `maxIndent` specifies how many times a block can be indented to the right, and the fact that blocks can be indented by 0 times is implicit; this means using `maxIndent` only results in an off by 1 with the rightmost indent line missing.